Source/FullScreenPokemon.ts

```diff
@@ -2511,6 +2524,8 @@ module FullScreenPokemon {
                     this.StateHolder.addChange(other.id, "sight", undefined);
                 }
             }
+
+            this.autoSave();
```

Comment:
Responded offline: per your feedback, we shouldn't be saving if a battle is about to start.